tabpy-server/tabpy_server/management/state.py

```diff
@@ -15,10 +15,8 @@
 
 from tabpy_server.management.util import write_state_config
 
-
```

Comment:
Not sure if there is intention here but deleting these lines will typically cause LINTing tools to fail